BUILD.md

```diff
@@ -106,6 +106,35 @@ can be listet by invoking `git tag -l`, where releases are indicated by a
 leading `v` followed by the release version number.
 
 
+Installing pre-commit hooks [optional]
+--------------------------------------
+Pre-commit hook is a good way to get feedback fast.
+Hooks implement a subset of checks that run on the CI, so it will jus save your time.
+
+Hooks work on all operating systems.
+You need to install them once, pre-commit will track config file and update itself in case of changes.
+
+- install git
+- install Python
+- [Install pre-commit](https://pre-commit.com/#install)
+    ```shell
+    pip install pre-commit
+    ```
+- install hooks from the project root
+    ```sh
+    pre-commit install
+    ```
+
+Notes:
+- Hooks might change your files, you will need to add (`git add`) changes to commit.
+- First commit after installation/update of hooks may take some time for installing tools.
+- You can skip checks by adding `-n, --no-verify` to `git commit`. This is useful when you commit partial done job.
+
+When you commit, hooks are run only on the changed files, if you want to run an all files:
+```shell
+pre-commit run --all-files
+```
```

Comment:
I don't have a strong opinion on this matter either. Let's wait maybe other people have preferences.

Reply:
@o01eg , @geoffthemedio , @Grummel7 , what is the place where you think the docs about Pyhton linting could be located?

Reply:
I don't have much context for an informed opinion, but I guess contributing sounds more relevant. CONTRIBUTING.md already mentions _The code changes match the [Code Standards].  For Python code we follow [PEP 8]._ so that seems similar?